app/controllers/api/articles_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Api::ArticlesController < ApplicationController
+  before_action :authenticate_admin!, only: [:create]
```

Comment:
for me this is not a enough, an editor should not be able to create an article, that is for the journalist. So just setting an admin model is not enough authorization in my opinion